Jenkinsfile

```diff
@@ -49,14 +49,19 @@ test ${TARGET_URL} --location us-east-1-linux:Chrome%20Canary --keepua  --noopt
         success {
           stash includes: 'wpt.json', name: 'wpt.json'
         }
-        failure {
-          ircNotification('#perftest-alerts')
-          emailext(
-            attachLog: true,
-            body: '$BUILD_URL\n\n$FAILED_TESTS',
-            replyTo: '$DEFAULT_REPLYTO',
-            subject: '$DEFAULT_SUBJECT',
-            to: '$DEFAULT_RECIPIENTS')
+      }
+    }
+    stage('Submit stats to Telemetry') {
+      agent {
+        dockerfile true
+      }
+      steps {
+        unstash 'wpt.json'
+        sh 'python ./send_to_telemetry.py wpt.json'
+      }
+      post {
+        success {
+          stash includes: 'wpt-telemetry-*.json', name: 'wpt-telemetry.json'
```

Comment:
Instead of stash, we should archive artifacts: https://jenkins.io/doc/pipeline/steps/core/#-archiveartifacts-%20archive%20the%20artifacts

Reply:
Oops yes.

Reply:
Fixed.

Reply:
Fixed.